pkg/master/master.go

```diff
@@ -407,11 +408,16 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 
 // InstallLegacyAPI will install the legacy APIs for the restStorageProviders if they are enabled.
 func (m *Master) InstallLegacyAPI(c *completedConfig, restOptionsGetter generic.RESTOptionsGetter, legacyRESTStorageProvider corerest.LegacyRESTStorageProvider) error {
-	legacyRESTStorage, apiGroupInfo, err := legacyRESTStorageProvider.NewLegacyRESTStorage(restOptionsGetter)
+	legacyRESTStorage, apiGroupInfo, destroyFunc, err := legacyRESTStorageProvider.NewLegacyRESTStorage(restOptionsGetter)
```

Comment:
surprised that you added a return value instead of using the same style of type-assertion as below.  Try to make these more alike, not more different.

Reply:
Which type-assertion style you have in mind? Can you point me to the line number?